lib/engine/game/g_1880/step/sell_company.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1880
+      module Step
+        class SellCompany < Engine::Step::Base
+          ACTIONS = %w[sell_company pass].freeze
+          def actions(entity)
+            return [] unless entity.player?
+            return [] unless can_sell?
+            return [] if sellable_companies(entity).empty? || @round.already_passed
+
+            ACTIONS
+          end
+
+          def auto_actions(entity)
```

Comment:
Why are auto actions needed? Don't we just need to prompt at the beginning of each SR or end of OR?

Reply:
If the private's award isn't claimed by a certain phase, it's auto-claimed. The auto_actions handles that use case